.github/workflows/lint.yml

```diff
@@ -1,7 +1,6 @@
 name: Lint
 
-on:
-  push:
+on: [push, pull_request]
```

Comment:
I would run either on `push` _or_ `pull_request`, since this is current running twice on PR commits.

![image](https://github.com/user-attachments/assets/d33a1c5b-dc9f-4be2-a943-e55151dbf99c)